cpp/log/log_signer.cc

```diff
@@ -142,6 +143,7 @@ LogSigner::SignResult LogSigner::SignV1TreeHead(uint64_t timestamp,
 }
 
 LogSigner::SignResult LogSigner::SignTreeHead(SignedTreeHead* sth) const {
+  LOG(INFO) << "Starting SignTreeHead for size: " << sth->tree_size();
```

Comment:
The changes in this file seem to be just general debugging, not really related to this change?